cabal-install/Distribution/Client/Get/Brancher.hs

```diff
@@ -0,0 +1,155 @@
+module Distribution.Client.Get.Brancher where
```

Comment:
I think we prefer explicit export lists. People introduce private utils during refactoring and shouldn't have to think about introducing an export list at the same time.